etc/config/functional.suite.dist.yml

```diff
@@ -16,6 +16,7 @@ modules:
         - \Magento\FunctionalTestingFramework\Helper\MagentoFakerData
         - \Magento\FunctionalTestingFramework\Module\MagentoSequence
         - \Magento\FunctionalTestingFramework\Module\MagentoAssert
+        - \Magento\FunctionalTestingFramework\Module\MagentoActionProxies
```

Comment:
@jilu1 Does this change affect the pipeline code since they have their own version of `functional.suite.yml`?

Reply:
From Ji in slack:

> Currently the functional.suite.dist.yml change is not used. In a later story, we will remove this dependency. I think I have a ticket number in the dry-run ticket for that work.